src/boilerplate.rs

```diff
@@ -437,6 +439,60 @@ fn add_local_terms(doc: &Spec, container: &NodeRef) {
     container.append(index_items_to_node(&index_items));
 }
 
+fn add_panels(doc: &Spec, container: &NodeRef) {
+    if let Ok(dfn_els) = doc.dom().select("dfn") {
```

Comment:
nit: I'd probably early-return, or just unwrap() as `"dfn"` is clearly a valid selector.

Reply:
You cleaned this up later so nvm